File: 1.C/8.c
/*22. Criar um programa que receba 3 notas de 3 alunos. Caso o aluno não obtenha nota superior a 6 na média
entre a duas notas AV1 e AV2, inserir nota AV3 e substituir a menor entre Av1 e Av2, recalcular a
média e mostrar o resultado da aprovação ou reprovação dos alunos*/

#include <stdio.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL, "Portuguese");

    double NOTAS[3][3];
    double MEDIAS[3];

    for (int i = 0; i < 3; i++)
    {
        printf("Aluno %d\n", i + 1);

        printf("AV1: ");
        scanf("%lf", &NOTAS[i][0]);
        printf("AV2: ");
        scanf("%lf", &NOTAS[i][1]);

        MEDIAS[i] = (NOTAS[i][0] + NOTAS[i][1]) / 2;

        if (MEDIAS[i] <= 6) 
        {
            printf("AV3: ");
            scanf("%lf", &NOTAS[i][2]);

            if (NOTAS[i][0] > NOTAS[i][1])
            {
                NOTAS[i][1] = NOTAS[i][2];
            }
            else
            {
                NOTAS[i][0] = NOTAS[i][2];
            }
            MEDIAS[i] = (NOTAS[i][0] + NOTAS[i][1]) / 2;
        }
    }

    for (int i = 0; i < 3; i++)
    {
        if (MEDIAS[i] > 6)
        {
            printf("Aluno %d: APROVADO\n", i + 1);
            printf("MÉDIA: %.2f\n", MEDIAS[i]);
        }
        else
        {
            printf("Aluno %d: REPROVADO\n", i + 1);
            printf("MÉDIA: %.2f\n", MEDIAS[i]);
        }
    }
}

File: 2.JS/8.js
/*31. Criar um programa que receba 3 NOTAS de 3 ALUNOS. Caso o aluno não obtenha nota superior a 6 na média
entre a duas NOTAS AV1 e AV2, inserir nota AV3 e substituir a menor entre Av1 e Av2, recalcular a
média e mostrar o resultado da aprovação ou reprovação dos ALUNOS*/

const alunos = 3;
const notas = [
    [7.0, 5.0],
    [4.0, 6.0],
    [6.5, 6.0]
];
const medias = [];

for (let i = 0; i < alunos; i++) {
    console.log(`Aluno ${i + 1}`);

    let av1 = notas[i][0];
    let av2 = notas[i][1];
    let av3 = 0;
    let media = (av1 + av2) / 2;

    if (media <= 6) {
        av3 = [3.0][0];
        if (av1 > av2) {
            av2 = av3;
        } else {
            av1 = av3;
        }
        media = (av1 + av2) / 2;
    }
    
    medias[i] = media;

    if (media > 6) {
        console.log(`Aluno ${i + 1}: APROVADO`);
    } else {
        console.log(`Aluno ${i + 1}: REPROVADO`);
    }
    console.log(`MÉDIA: ${media.toFixed(2)}`);
}

